packages/core/lib/core.dart

```diff
@@ -2,5 +2,4 @@
 library core;
 
 export './src/styles/styles.dart';
-export './src/tweet/tweet.dart';
```

Comment:
Looks odd to me that this was removed. Is this intentional?

Reply:
Yes. It's now inside `src/widgets/widgets.dart` 
I have some concerns in general for the folder structure that I would love to discuss with you.